ansible_collections/arista/avd/roles/eos_validate_state/tasks/loopback_reachability.yml

```diff
@@ -0,0 +1,56 @@
+---
+
+- name: Gather ip reachability state between devices (Router ID <-> Router ID)
+  eos_command:
+    commands: "ping {{ loopback_ip_address | ansible.netcommon.ipaddr('address') }} source {{ loopback_interfaces[router_bgp.peer_groups[\"EVPN-OVERLAY-PEERS\"].update_source].ip_address | ansible.netcommon.ipaddr('address') }} repeat 1"
```

Comment:
Points for creativity, but this will not work. The peer group name is configurable.
I really think we should just add a variable for this